### DISCUSSION TITLE
Scoop installation error Win10 PowerShell 5.1...

### DISCUSSION THREAD

**Discussion by GrossmanYaad** [ScoopInstaller/Scoop#4787] [Help and Questions] [answered]
When using the following command in PowerShell : 
Invoke-Expression (New-Object System.Net.WebClient).DownloadString('https://get.scoop.sh/')
I get the following error:
_Invoke-Expression : The term 'Select-CurrentVersion' is not recognized as the name of a cmdlet, function, script file,
or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and
try again.
At line:1 char:1
+ Invoke-Expression (New-Object System.Net.WebClient).DownloadString('h ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Select-CurrentVersion:String) [Invoke-Expression], CommandNotFoundExcep
   tion
    + FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.InvokeExpressionCommand_
![image](https://user-images.githubusercontent.com/93009691/157239405-68a762d4-9ef3-405f-8aa3-93194643e59a.png)

**Any ideas here ?**

**Comment by rashil2000** [comment, marked as answer]
Fix incoming: https://github.com/ScoopInstaller/Scoop/pull/4785

**Reply by GrossmanYaad** [reply]
Is there anything I can do to install scoop now ? When will the fix be implemented ?

**Reply by rashil2000** [reply]
See https://github.com/ScoopInstaller/Scoop/issues/4783#issuecomment-1061446902

**Reply by GrossmanYaad** [reply]
Thank you Rashil Gandhi. I wish you the best.

**Reply by rashil2000** [reply]
The fix has been merged
